providers/src/airflow/providers/amazon/aws/operators/emr.py

```diff
@@ -636,8 +641,14 @@ class EmrCreateJobFlowOperator(BaseOperator):
     :param job_flow_overrides: boto3 style arguments or reference to an arguments file
         (must be '.json') to override specific ``emr_conn_id`` extra parameters. (templated)
     :param region_name: Region named passed to EmrHook
-    :param wait_for_completion: Whether to finish task immediately after creation (False) or wait for jobflow
+    :param wait_for_completion: Deprecated - use `wait_policy` instead.
+        Whether to finish task immediately after creation (False) or wait for jobflow
         completion (True)
+        (default: None)
+    :param wait_policy: Whether to finish the task immediately after creation (None) or:
+        - wait for the jobflow completion (WaitPolicy.WAIT_FOR_COMPLETION)
+        - wait for the jobflow completion and cluster to terminate (WaitPolicy.WAIT_FOR_STEPS_COMPLETION)
+        (default: None)
     :param waiter_max_attempts: Maximum number of tries before failing.
```

Comment:
Can you add the explnation about the parameter also to the emr docs?
Also lets verify if we have example dags using the old parameter and switch them to the new one

Reply:
> Also lets verify if we have example dags using the old parameter and switch them to the new one

I did check and we do not :)

Reply:
No examples using the old parameter. 

I've updated the doc to describe `wait_policy`